frontend/src/layout/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -1,16 +1,30 @@
 import './ErrorBoundary.scss'
 
 import { ErrorBoundary as SentryErrorBoundary, getCurrentHub } from '@sentry/react'
-import { useActions } from 'kea'
+import { useActions, useValues } from 'kea'
 import { supportLogic } from 'lib/components/Support/supportLogic'
 import { LemonButton } from 'lib/lemon-ui/LemonButton'
+import { teamLogic } from 'scenes/teamLogic'
 
-export function ErrorBoundary({ children }: { children?: React.ReactNode }): JSX.Element {
+export function ErrorBoundary({
+    children,
+    tags = {},
+}: {
+    children?: React.ReactNode
+    tags?: Record<string, any>
+}): JSX.Element {
     const isSentryInitialized = !!getCurrentHub().getClient()
+    const { currentTeamId } = useValues(teamLogic)
     const { openSupportForm } = useActions(supportLogic)
 
     return (
         <SentryErrorBoundary
+            beforeCapture={(scope) => {
+                if (currentTeamId) {
+                    scope.setTag('team_id', currentTeamId)
+                }
+                Object.entries(tags).map(([key, value]) => !!value && scope.setTag(key, value))
```

Comment:
This value check won't allow using a zero or other nullable values like false. It's better to only check for the `undefined` if you want to prevent unsetting. Otherwise, I'd use `Sentry.setTags` with the `tags` object.

`forEach` also makes more sense here because you don't need a returned array.

Reply:
Thanks for pointing this out! Forgot to update to `forEach` 🤦🏽, but using setTags now which is much cleaner